### DISCUSSION TITLE
Scrypted LXC with Coral USB TPU does not work

### DISCUSSION THREAD

**Discussion by skynan22** [tteck/Proxmox#1802] [Help Needed]
I have two Coral USB TPUs. First is connected with Frigate LXC and work fine the second (new) one should be connected with Scrypted LCX for my own object detection project. The second one does not work (so far).

What I did:
- lsusb on pve host shell:
`Bus 004 Device 010: ID 1a6e:089a Global Unichip Corp. `

- Added one line in my 101.conf (<- Scrypted LXC) from pve shell :
  `lxc.mount.entry: /dev/bus/usb/010 dev/bus/usb/010 none bind,optional,create=dir 0, 0`

- Followed install instructions within Scrypted Shell from [Coral](https://coral.ai/docs/accelerator/get-started/#requirements)

Scrypted LXC is based on Ubuntu (error?) I can not install the example from Coral because of Python V3.10.
Scrypted LXC does not recognize USB Stick as "Google" but stills shows "Global Unichip Corp." as described above.
I have no proof, but after plugging out and attaching the stick again, the number at the end switched from 09 -> 10 (now) while 101.conf still adresses 09.

Is there a way to get a second USB TPU to work as described above?

**Comment by marleyjaffe** [comment]
I have a very different situation, but my USB Coral device changes from Global Unichip to Google after it initializes with the Operating system. I use a passthrough a USB PCIe Card instead of the individual USB to solve this for me.
